### PR TITLE
[FIX] [16.0] hr_contract_update_overtime: Reorganize Leaves across Contracts + Message

### PR DESCRIPTION
**Pull request by Shide**
- Note when Updating Overtime
- Reorganize leaves across Calendars when Update Overtime from Contract History

MT-6583 @moduon  @rafaelbn @loida-vm @fcvalgar please reivew if you want 😄

### CODE REVIEW

**Comment by rafaelbn** [comment]
Ping me when I can test it @Shide ! ❤️ Thank you! 😄

**Review by rafaelbn — APPROVED** [review]
Functional tested in runboat! 👍🏼 

Just missing fix tests @yajo

File: hr_contract_update_overtime/README.rst [hunk 1]
```diff
@@ -38,6 +38,9 @@ You can Update Overtime in different ways:
 -  Header button on Contract History form view
 -  Header button on Contract form view
```
Comment (fcvalgar) [inline hr_contract_update_overtime/README.rst:39]: - Header button on Contract form view is visible if the user is responsible HR for the employee

**Review by fcvalgar — APPROVED** [review]
Functional review.

LGTM, great @Shide 

[OCA | [FIX] [16.0] hr_contract_update_overtime: Reorganize Leaves across Contracts + Message #172 - Watch Video](https://www.loom.com/share/b6904b2fff0b47a7a0331d60c9120c6d)

**Review by yajo — APPROVED** [review]
I fixed the tests here and it should be ready to merge, but I cannot take it out from draft. Could you do it please @Shide?

**Comment by Shide** [comment]
> I fixed the tests here and it should be ready to merge, but I cannot take it out from draft. Could you do it please @Shide?

Done

**Pull request merged**